### PR TITLE
Deployed for Juniors FFCS 

### PR DESCRIPTION
**Pull request by yashhatwar**
Made changes and deployment is successfull.Updated for FFCS of 2022 Juniors.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
         }
     },
     "engines": {
-        "node": ">=10 <13"
+        "node": ">=16"
```
Comment (vatz88) [inline package.json:26]: change wasn't needed

```suggestion
        "node": ">=10 <13"
```

**Comment by vatz88** [comment]
@sanket9918 can you please verify

**Comment by vatz88** [comment]
@yashhatwar thanks for making efforts
Can you please test this https://deploy-preview-212--ffcsonthego.netlify.app/

Also, does this have all courses for summer semester 2022?

**Comment by yashhatwar** [comment]
No actually I had to update the excel once again,will do it in a minute and then its good to go

**Comment by vatz88** [comment]
@yashhatwar can you please confirm if this sheet is for `summer semester 2022`? And if it has all courses?

**Comment by yashhatwar** [comment]
Yes now the updated one has all the courses

**Comment by vatz88** [comment]
Thanks @yashhatwar !

Can you also please update it here https://github.com/vatz88/FFCSonTheGo/blob/master/package.json#L5

**Comment by vatz88** [comment]
@yashhatwar also please do test this https://deploy-preview-212--ffcsonthego.netlify.app/

**Comment by yashhatwar** [comment]
Tested!!! its Working

**Comment by sanket9918** [comment]
> @sanket9918 can you please verify

Sorry for the delay. Checked the final deployment and it's working great. Thanks @yashhatwar!

**Pull request merged**